src/parsimmon.js

```diff
@@ -420,6 +420,13 @@
     });
   };
 
+  _.else = function(result) {
+    return this.or(succeed(result));
+  };
+
+  //- ES3 compatibility
+  _.else_ = _.else;
+
```

Comment:
For ES3 compatibility we'll need this as `_["default"] = function...` and then `_.default_ = _["default"]`